.gitignore

```diff
@@ -0,0 +1,4 @@
+*.egg-info/
+*.pyc
+.coverage
+
```

Comment:
Including a `.gitignore` with this PR might not be desired based on https://github.com/pypa/distutils/pull/45#issuecomment-873626642

Reply:
Thanks for the context! Removed the file; I'm fine with a global git-ignore configuration. Apologies for the force push.